src/main/scala/arcana/death/Repl.scala

```diff
@@ -0,0 +1,29 @@
+package arcana.world
+
+import scala.io.StdIn
+
+object Repl {
```

Comment:
I still think that making REPL a class rather than an object is a better choice:
- Classes can take arguments in the constructor
- Even though only one REPL would exist at the same time, there still exists the situation where we don't launch the REPL. Defering the REPL initialization lazily may be a better choice